test/tools/router_check/router.cc

```diff
@@ -40,18 +41,13 @@ ToolConfig::ToolConfig(std::unique_ptr<Http::TestHeaderMapImpl> headers, int ran
     : headers_(std::move(headers)), random_value_(random_value) {}
 
 // static
-RouterCheckTool RouterCheckTool::create(const std::string& router_config_json) {
+RouterCheckTool RouterCheckTool::create(const std::string& router_config_yaml) {
   // TODO(hennna): Allow users to load a full config and extract the route configuration from it.
-  Json::ObjectSharedPtr loader = Json::Factory::loadFromFile(router_config_json);
   envoy::api::v2::RouteConfiguration route_config;
-  // TODO(ambuc): Add a CLI option to allow for a maxStatNameLength constraint
-  Stats::StatsOptionsImpl stats_options;
-  Config::RdsJson::translateRouteConfiguration(*loader, route_config, stats_options);
+  MessageUtil::loadFromFile(router_config_yaml, route_config);
```

Comment:
IIUC we're removing support for v1 configurations. can you include that in the version_history and make that clear in the docs?

Reply:
Took care of mentioning it in the version_history file, but I didn't feel we needed to update the docs since the schema it links to is the v2 route schema.